longevity_test.py

```diff
@@ -171,6 +180,14 @@ def test_custom_time(self):
         for stress in stress_queue:
             self.verify_stress_thread(queue=stress)
 
+        # if validate_partitions:
+        #     self.log.debug('Save partitons info after reads')
+        #     partitions_dict_after = self.collect_partitions_info(table_info=validate_partitions,
+        #                                                          save_into_file_name='partitions_rows_after.log')
```

Comment:
why this commented out code is here?

Reply:
This is new part that was asked by Shlomi. I did it but didn't test still because of the test was changed so that this part is not run. I'll discuss with Roy about this